v1.0/conformance_test_v1.0.yaml

```diff
@@ -2590,3 +2590,38 @@
   doc: Test an anonymous enum inside an array inside a record, SchemaDefRequirement
   tags: [command_line_tool, schema_def]
 
+- tool: v1.0/loadContents-output-contains-b.cwl
+  job: v1.0/64kbplus1-job.json
+  output:
+    contains_b: false
```

Comment:
based on conversations in the room at CWL meeting 1/24 in Boston - we've decided this is the worst of all behaviors and we'd like to fail here.